src/components/ItemList.svelte

```diff
@@ -54,13 +54,16 @@
 
 <style>
   .item-browser {
-    max-height: 400px;
-    overflow: scroll;
     a {
       text-decoration: none;
     }
   }
 
+  .item-property {
+    height: 36px;
+    overflow: hidden;
+  }
```

Comment:
Let's make the div a little bit taller. Also, if we set `overflow-y` to be auto here the content is still hidden when overflown, but with a scrollbar visible for users to read the rest of the content

```suggestion
  .item-property {
    height: 40px;
    overflow-y: auto;
  }
```